src/docs/markdown/running.md

```diff
@@ -152,6 +156,23 @@ RestartSec=5s
 Then, save the file and exit the text editor, and restart the service for it to take effect:
 <pre><code class="cmd bash">sudo systemctl restart caddy</code></pre>
 
+### SELinux System Considerations
+
+On SELinux enabled systems you have two options:
+1. Compile Caddy and label the files yourself
+2. Install Caddy using the [COPR repo](https://copr.fedorainfracloud.org/coprs/g/caddy/caddy/), your systemd file and caddy binary will be created and labelled correctly. NB: if you want to recompile Caddy with modules, you will have to label the executable as described bellow.
+   
+
+Systemd unit files and their executables will not be run unless labelled with `systemd_unit_file_t` and `bin_t` respectively.
+
+The `systemd_unit_file_t` is automatically applied to files created in `/etc/systemd/...`, so be sure to create your `caddy.service` file there.
+
+To tag the caddy binary, you can use the following commands:
+```shell
```

Comment:
Please take a look at how we're doing code blocks for commands, i.e. HTML so that it has the CSS classes we want to have, `cmd bash`

Reply:
Thanks, will push in the next commit !